UnityProject/Assets/Scripts/Managers/SubSceneManager/SubSceneManager.SceneList.cs

```diff
@@ -243,41 +232,42 @@ public IEnumerator LoadSyndicate()
 	{
 		if (SyndicateLoaded) yield break;
 		var pickedMap = additionalSceneList.defaultSyndicateScenes.PickRandom();
-
+		/*
 		foreach (var syndicateData in additionalSceneList.SyndicateScenes)
 		{
 			if (syndicateData.DependentScene == null || syndicateData.SyndicateSceneName == null)
 				continue;
 			if (syndicateData.DependentScene != serverChosenMainStation)
 				continue;
 
-			pickedMap = syndicateData.SyndicateSceneName;
+			pickedMap = syndicateData;
 			break;
 		}
+		*/
```

Comment:
Won't have the specific syndicate scenes anymore? This is fine so can remove this commented code, wasnt a big feature anyway.

Reply:
Going to remove comments soon after I'm done with testing and working on everything else